KubeArmor/core/kubeUpdate.go

```diff
@@ -190,6 +190,7 @@ func (dm *KubeArmorDaemon) UpdateEndPointWithPod(action string, pod tp.K8sPod) {
 
 		newPoint.NamespaceName = pod.Metadata["namespaceName"]
 		newPoint.EndPointName = pod.Metadata["podName"]
+		newPoint.DeploymentName = pod.Metadata["deploymentName"]
```

Comment:
We need to handle deployments, replicasets, statefulsets etc.
Currently, only deployment name is handled. I think the issue explains possible way to handle it.

Reply:
updated the pr with owner information